### PR TITLE
fix: crash with `InvalidAuthBlockingTokenError`

### PR DESCRIPTION
**Pull request by exaby73**
Fixes https://github.com/firebase/firebase-functions-python/issues/133
Fixes https://github.com/firebase/firebase-functions-python/issues/144

### CODE REVIEW

File: src/firebase_functions/private/_identity_fn.py [hunk 1]
```diff
@@ -64,11 +64,14 @@ def _auth_user_info_from_token_data(token_data: dict[str, _typing.Any]):
 
 def _auth_user_metadata_from_token_data(token_data: dict[str, _typing.Any]):
     from firebase_functions.identity_fn import AuthUserMetadata
+    creation_time = _dt.datetime.utcfromtimestamp(int(token_data["creation_time"]) / 1000.0)
+    last_sign_in_time = None
+    if "last_sign_in_time" in token_data:
+        last_sign_in_time = _dt.datetime.utcfromtimestamp(int(token_data["last_sign_in_time"]) / 1000.0)
```
Comment (russellwheatley) [inline src/firebase_functions/private/_identity_fn.py:70]:
```suggestion
        last_sign_in_time = _dt.datetime.utcfromtimestamp(int(token_data["last_sign_in_time"] / 1000.0 ))
```
Think it should be this.

**Review by russellwheatley — CHANGES_REQUESTED** [review]

**Review by russellwheatley — APPROVED** [review]
LGTM

**Comment by exaby73** [comment]
Lint error will be fixed on merge of https://github.com/firebase/firebase-functions-python/pull/147

**Review by Salakar — APPROVED** [review]

**Comment by taeold** [comment]
Before I merge - can you fix the linter issues?

**Comment by EmilioNicolas** [comment]
Hi! When can we expect this issue merged and deployed? thanks!!

**Comment by EmilioNicolas** [comment]
Hi! Is it already in PRO? should we wait to other release?

**Comment by exaby73** [comment]
This should be fixed in 0.1.2, which hasn't been released yet, but will be soon

**Pull request merged**